src/jit/lsraarmarch.cpp

```diff
@@ -559,20 +559,18 @@ void Lowering::TreeNodeInfoInitCall(GenTreeCall* call)
         // Skip arguments that have been moved to the Late Arg list
         if (!(args->gtFlags & GTF_LATE_ARG))
         {
+#ifdef DEBUG
+            fgArgTabEntryPtr curArgTabEntry = compiler->gtArgEntryByNode(call, arg);
+            assert(curArgTabEntry);
+#endif
             if (arg->gtOper == GT_PUTARG_STK)
             {
-                fgArgTabEntryPtr curArgTabEntry = compiler->gtArgEntryByNode(call, arg);
-                assert(curArgTabEntry);
-
-                assert(curArgTabEntry->regNum == REG_STK);
+                INDEBUG(assert(curArgTabEntry->regNum == REG_STK));
```

Comment:
Why do these asserts need to be surrounded by `INDEBUG`? `assert` is always DEBUG only.

Reply:
Doh! Thanks, I'll remove the `INDEBUG`.